x-pack/plugins/fleet/server/collectors/fleet_server_collector.ts

```diff
@@ -39,16 +39,8 @@ export const getFleetServerUsage = async (
     return DEFAULT_USAGE;
   }
 
-  const numHostsUrls = await settingsService
-    .getSettings(soClient)
-    .then((settings) => settings.fleet_server_hosts?.length ?? 0)
-    .catch((err) => {
-      if (isBoom(err) && err.output.statusCode === 404) {
-        return 0;
-      }
-
-      throw err;
-    });
+  const fleetServerHosts = await listFleetServerHosts(soClient);
+  const numHostsUrls = fleetServerHosts.total;
```

Comment:
I update this telemetry collector to use the new Fleet Server Hosts model for its calculation instead of the settings SO.

Reply:
Good catch 👍